.github/workflows/build.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Setup LXD
-        uses: canonical/setup-lxd
+        uses: canonical/setup-lxd@v0
```

Comment:
there are no tag. You have to use branch `main`
https://github.com/canonical/setup-lxd/branches

Reply:
the canonical/setup-lxd@main takes 15 min (vs 11 secs) as it tries to update lxd. 

We agreed with @taurus-forever to stick to the workaround with `iptables` (and whywaita/setup-lxd@v1) for the time being, given the substantial improvement in CI/CD pipeline

Reply:
@taurus-forever @deusebio: I've updated `canonical/setup-lxd` so it won't refresh LXD if you don't give a channel. So
```
uses: canonical/setup-lxd@v0.1.0
```
should work for you.